src/styles/animations.css

```diff
@@ -29,3 +29,33 @@
 	animation-fill-mode: both;
 	animation-name: fadeOut;
 }
+
+@keyframes slideInDown {
+	from {
+		max-height: 0;
+	}
+	to {
+		max-height: 300px;
+	}
+}
+
+.slideInDown {
```

Comment:
I think we need to change this name, in `animate-js` this is the class / naming for sliding something *in* and *down* from off the page. I think this should be `slideDown` or `expandDown` We likely need to be cleverer than this though as we want to make sure the text doesn't move about as we expand and it may be better to fade that in at the same time? Worth playing with a little. Also, where did `300px` come from for a max height? Seems restrictive, what are other frameworks / solutions doing?